source/predicates/string.ts

```diff
@@ -127,6 +127,16 @@ export class StringPredicate extends Predicate<string> {
 		});
 	}
 
+	/**
+	Test a string to be not empty nor only whitespace.
+	 */
+	get nonBlank(): this {
+		return this.addValidator({
+			message: (_, label) => `Expected ${label} to not be blank`,
```

Comment:
You need to include what it got.

Reply:
Outputting a string of whitespace seems disruptive. Would you like quote marks around it, or a character count instead?

Reply:
Ah nvm, I see in the other validators the value is enclosed in `` ` ``

Reply:
Probably still not super helpful to output whitespace though. Perhaps we should replace common whitespace like `\n` and `\t` by their literals, and spaces by `␠`?

Reply:
Yeah, lets use the glyphs.

Reply:
Isn't the poor legibility and low chance of anyone recognizing their meaning on sight an issue?

Reply:
I don't really have a strong opinion. It won't come up often anyway.

I'm fine with "Perhaps we should replace common whitespace like \n and \t by their escape strings, and spaces by ␠/⎵/·?" too.